nucypher/utilities/sandbox/blockchain.py

```diff
@@ -89,7 +89,7 @@ def __generate_insecure_unlocked_accounts(self, quantity: int) -> List[str]:
 
             umbral_priv_key = UmbralPrivateKey.gen_key()
             address = self.interface.w3.personal.importRawKey(private_key=umbral_priv_key.to_bytes(),
-                                                              password=insecure_password)
+                                                              passphrase=insecure_password)
```

Comment:
Not related to this PR, but do we still consider `"passphrase"` in Sentry scrubbers?

Reply:
Yes.